### ISSUE TITLE
How to refresh auth state in main window after popup authorization

### ISSUE THREAD

**Issue by cam92yz** [damienbod/angular-auth-oidc-client#931]
I want to use popup authorization and use following steps:

1. Open popup with authorizeWithPopUp()
2. Close popup window in callback and send event to the main Window
3. Problem: The token on the main Window is not refreshed. (It works with reaload but i don't want to do that because i don't want to loose the state of the window)

According to "https://github.com/damienbod/angular-auth-oidc-client/issues/212" there was a method "authorizedCallback(theHash)" that handled this problem. This method was renamed to "authorizedImplicitFlowCallback" and was later deleted.

I use the version "11.3.0". How can i refresh the state without reloading?

**Comment by cam92yz** [comment]
Anyone an idea? Have to solve that problem in our application.

**Comment by damienbod** [comment]
@FabianGosebrink I think we should add a popup example to the repo.

Popup code flow with PKCE

good?

**Comment by FabianGosebrink** [comment]
Hey, can we build that into the code flow sample we already have and then document it?

**Comment by damienbod** [comment]
I would prefer a separate example so that it's crystal clear. I would be afraid people might get confused if we mix two different examples together. good?

**Comment by otrebor-bni** [comment]
@damienbod @FabianGosebrink 
whats the actual state? We would be happy to have an working sample with a popup codeflow too.

**Comment by FabianGosebrink** [comment]
We are currently working on the next release and after this, this issue is one of the things we tackle next. We are accepting PRs by the way :-) We will let you know if we made code changes. @damienbod another sample is good for me. but maybe only a part of the issue. We have to make sure that what @otrebor-bni said: The token on the main Window is not refreshed. So two tasks here: 

1) Creating the sample
2) Maybe the fix for the original issue (I did not check it so far)

Thanks.
